jwst/lib/set_telescope_pointing.py

```diff
@@ -148,6 +155,100 @@ def update_wcs(model, default_pa_v3=0., siaf_path=None, **kwargs):
         Keyword arguments used by matrix calculation routines.
     """
 
+    # If the type of exposure is not FGS, then attempt to get pointing
+    # from telemetry.
+    try:
+        exp_type = model.meta.exposure.type.lower()
+    except AttributeError:
+        exp_type = None
+    if exp_type in FGS_GUIDE_EXP_TYPES:
+        update_wcs_from_fgs_guiding(
+            model, default_pa_v3=default_pa_v3
+        )
+    else:
+        update_wcs_from_telem(
+            model, default_pa_v3=default_pa_v3, siaf_path=siaf_path, **kwargs
+        )
+
+
+def update_wcs_from_fgs_guiding(model, default_pa_v3=0.0, default_vparity=1):
+    """ Update WCS pointing from header information
+
+    For Fine Guidance guidine observations, nearly everything
+    in the `wcsinfo` meta information is already populated,
+    except for the CD matrix. This function updates the CD
+    matrix based on the rest of the `wcsinfo`.
+
+    Parameters
+    ----------
+    model : `~jwst.datamodels.DataModel`
+        The model to update.
+
+    default_pa_v3 : float
+        If pointing information cannot be retrieved,
+        use this as the V3 position angle.
+
+    default_vparity: {-1, 1}
```

Comment:
I'm not sure how sphinx will parse this as it is supposed to be a type. Explain the possible values in the definition. 
Also use double quotes as in ``1`` , otherwise sphinx will attempt to link to it.